packages/react-scripts/template/index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module '*.jpg'
```

Comment:
I don't think this file should be called `index.d.ts` as this name will conflict with the automatically generated typings file for `index.ts`. In my projects I normally call this file `externals.d.ts` or similar but I don't really mind what it is called.